docs/es/advanced/lazy-loading.md

```diff
@@ -0,0 +1,44 @@
+# Carga diferida de rutas
+
+Cuando se construyen aplicaciones con un sistema de empaquetamiento de módulos, el archivo JavaScript resultante puede terminar siendo muy grande, afectando los tiempos de carga de la página. Sería más eficiente si pudiesemos dividir los componentes de cada ruta en porciones separadas y cargarlas solo cuando esa ruta es visitada.
```

Comment:
Yo prefiero guardar los términos que habéis elegido. Me fijo más en que lo que se dice sea correcto para vue-router y vue y algunás que otras frases que suenan raras